.github/workflows/lint.yml

```
@@ -0,0 +1,41 @@
+name: Lint Hacktion
+
+on: 
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs: 
+  lint:
+    name: Run linters
+    runs-on: ubuntu-latest
+    steps:
+      # Paso 1: Obtener el código del repositorio
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Paso 2: Configurar Deno en el runner
+      - name: Setup Deno
+        uses: denoland/setup-deno@v2
+        with:
+          deno-version: v2.x
+          
+      - name: Install packages
+        run: deno install
+      # Paso 4: Ejecutar ESLint para verificar el código
+      - name: Run lint
+        run: deno lint
+        
+      # Paso 5: Verificar la configuración de Astro
+      - name: Check Astro
+        run: deno run --allow-read --allow-run --allow-env --allow-net npm:astro check
```

```suggestion
        run: deno run -A npm:astro check
```
Para ejecución con todos los permisos, estamos seguros que el paquete es el correcto